src/playback/qml/MuseScore/Playback/MixerPanel.qml

```diff
@@ -68,6 +68,17 @@ ColumnLayout {
         }
     }
 
+    function scrollToFocusedItem(focusedIndex) {
+        let targetScrollPosition = (focusedIndex) * (prv.channelItemWidth + 2)
+        let maxContentX = flickable.contentWidth - flickable.width
+
+        if (targetScrollPosition + prv.channelItemWidth > flickable.contentX + flickable.width) {
+            flickable.contentX = Math.min(targetScrollPosition + prv.channelItemWidth - flickable.width, maxContentX)
+        } else if (targetScrollPosition < flickable.contentX) {
+            flickable.contentX = Math.max(targetScrollPosition - 90, 0)
```

Comment:
Where does `90` come from here?

Reply:
@mathesoncalum I guessed this earlier based on debugging some values, but now updated correctly.

Reply:
That's great, thank you.

If you could now please [squash this PR](https://musescore.org/en/handbook/developers-handbook/finding-your-way-around/git-workflow#squash) into 1 or 2 commits that would be ideal. We can address the properties panel in a separate PR.

Reply:
@mathesoncalum Done. @cbjeukendrup, thanks for [this](https://github.com/musescore/MuseScore/pull/25664#issuecomment-2510031056). I’ll definitely look into it this and raise a PR for properties panel whenever possible.